### PR TITLE
Add database and user for postgres

### PR DESCRIPTION
**Pull request by vibindaniel**
# Description

Add database and user
Also added pre-commit to add changelog

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

tf validate

- [ ] Sanity Testing

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added documentation to test

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  special = false
+}
+
+resource "postgresql_role" "my_role" {
+  provider = postgresql.this
+
+  name            = var.role_name
+  login           = true
+  password        = random_password.role_password.result
+  create_database = true
```
Comment (LeeHughsVariant) [inline main.tf:12]: should this not be var.create_database?

File: README.md [hunk 2]
```diff
@@ -1,9 +1,58 @@
-# Terraform Postgres Database
+# Postgres Terraform Module
```
Comment (LeeHughsVariant) [inline README.md:1]: Auto generated README is nice, but can we give engineering team a little better explanation of how to use the module? Namely concerned with setting up the provider. Will use of this module will deprecate the current confluence page?

File: main.tf (hunk 1 above)
Comment (vibindaniel) [reply main.tf]: fixes

File: README.md (hunk 2 above)
Comment (vibindaniel) [reply README.md:1]: Added examples

**Review by salyateem13 — APPROVED** [review]

**Pull request merged**